src/client/pages/Offer/Introduction/Sidebar/StartDate.tsx

```diff
@@ -307,18 +301,25 @@ export const StartDate = ({
 }: StartDateProps) => {
   const textKeys = useTextKeys()
   const [showError, setShowError] = useState(false)
-  const [loadingQuoteIds, setLoadingQuoteIds] = React.useState<Array<string>>(
-    [],
-  )
+  const [loadingQuoteIds, setLoadingQuoteIds] = useState<Array<string>>([])
 
   const handleSelectNewStartDate = async (
     newDateValue: Date | null,
     quoteIds: Array<string>,
   ) => {
     try {
       setShowError(false)
-      setLoadingQuoteIds(quoteIds)
-      await onSelect(newDateValue, quoteIds)
+
+      const quotesToBeUpdated = [...quoteIds]
+      const selectedCarQuote = selectedQuotes.find(isCar)
```

Comment:
I don't think you need to find this, can't you just send all `quotesToBeUpdated`? Or will `selectedQuotes` contain even more quotes than `quoteIds`?

Reply:
You're right. `selectedQuotes` will contain the quotes of the selected bundle variant. So this code is faulty.